README.md

```
@@ -182,6 +182,14 @@ revolved = nurbs.create.revolved(
 splinepy.show(["extruded", extruded], ["revolved", revolved])
 ```
 ![create_derived](docs/source/_static/readme_create_derived.png)
+You can also create swept surfaces and solids along spline trajectories.
+```python
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)